modules/citeproc/includes/marcrelator_conversion.inc

```diff
@@ -5,6 +5,7 @@
  * This is a mapping for translation from mods to json.
  */
 
+global $_ir_citation_marcrelator_codes; 
```

Comment:
@jefferya there is a whitespace at the end of this line. That makes phpcs fail on travis.